package.json

```diff
@@ -60,6 +60,17 @@
     "pnpm": ">=9",
     "node": ">=20"
   },
-  "keywords": ["postgres", "firebase", "storage", "functions", "database", "auth"],
-  "packageManager": "pnpm@9.15.5"
+  "keywords": [
+    "postgres",
+    "firebase",
+    "storage",
+    "functions",
+    "database",
+    "auth"
+  ],
+  "packageManager": "pnpm@9.15.5",
+  "dependencies": {
+    "@supabase/shared-types": "0.1.80",
```

Comment:
These imports are probably wrong?

Reply:
you're right - these shouldn't be here sorry